ui/page/components/items_scroll.go

```diff
@@ -58,7 +60,8 @@ func NewScroll[T any](load *load.Load, pageSize int32, queryFunc ScrollFunc[T])
 
 // FetchScrollData is a mutex protected fetchScrollData function. At the end of
 // the function call a window reload is triggered. Returns that latest records.
-func (s *Scroll[T]) FetchScrollData(isReverse bool, window app.WindowNavigator) {
+// If reset is true, the offset value will be reset to 0.
+func (s *Scroll[T]) FetchScrollData(reset bool, window app.WindowNavigator) {
 	s.mu.Lock()
 	// s.data is not nil when moving from details page to list page.
 	if s.data != nil {
```

Comment:
What does this comment mean? What's happening inside this if statement?
It's unclear code like these that leads to bugs.

If you fully understand all the code in this file, please add generous comments in as many places as possible.
If you don't fully understand what's happening here, then you shouldn't be maintaining or improving it. We should probably be talking about a new struct that achieves the intended objectives, but is much clearer and easier to read and understand.